doozer/doozerlib/cli/scan_osh.py

```diff
@@ -148,8 +148,11 @@ def check_if_scan_issues_exist(self, task_id: str, nvr: str) -> bool:
         self.runtime.logger.info(f"Checking OSH Scan for issues: {url}")
         response = requests.get(url)
 
-        if len(response.json()["defects"]) > 0:
-            return True
+        try:
+            if len(response.json()["defects"]) > 0:
```

Comment:
I think we want to include requests.get line inside try as well?

Reply:
Hmm I thought that line will never fail, like it will have a failed status code but it would break the code right.

But updated nevertheless https://github.com/openshift-eng/art-tools/pull/198/commits/6af2210ba3e775168f1f2aae7689612c1aa474bd